keyboards/lets_split_eh/keymaps/resfury/keymap.c

```diff
@@ -0,0 +1,194 @@
+  _______, KC_7,    KC_8,  KC_9,   KC_ASTR, KC_SLSH, KC_ASTR, KC_SLSH, KC_DOT,  KC_MUTE, RCTL(KC_BSLS), KC_BSLS, \
+  _______, KC_COMM, KC_0,  KC_DOT, ADJUST,  KC_BSPC, KC_SPC,  _______, KC_MPLY, KC_VOLU, KC_VOLD,       KC_MNXT  \
+),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |Taskmg|      |      |      |      |      |      |      |RGBVAI|RGBSAI|RGBHUI|caltde|
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |_COLEMAK|    |      |      |      |      |      |      |RGBVAD|RGBSAD|RGBHUD|RGBTOG|
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |_DVORAK|     |      |      |      |      |      |      |      |      |RGBMOD|BLSTEP|
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |_QWERTY|     |      |      |      |      |      |      |      |      |      | RESET|
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT( \
+  TSKMGR,       _______, _______, _______, _______, _______, _______, _______, RGB_VAI, RGB_SAI, RGB_HUI, CALTDEL, \
+  DF(_COLEMAK), _______, _______, _______, _______, _______, _______, _______, RGB_VAD, RGB_SAD, RGB_HUD, RGB_TOG, \
+  DF(_DVORAK),  _______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_MOD, BL_STEP, \
+  DF(_QWERTY),  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RESET    \
+),
+
+/* Function
+ * ,-----------------------------------------------------------------------------------.
+ * | Caps |      |      |      |      |      | S(Hm)| Home |  Up  | End  |S(End)|      |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      | Ctrl | Shift| Alt  |      |      |      | Left | Down |Right |      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_FUNCTION] = LAYOUT( \
+  KC_CAPS, _______, _______, _______, _______, _______, S(KC_HOME), KC_HOME, KC_UP,   KC_END,  S(KC_END), _______, \
+  _______, KC_LCTL, KC_LSFT, KC_LALT, _______, _______, _______,    KC_LEFT, KC_DOWN, KC_RGHT, _______,   _______, \
+  _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______, _______,   _______, \
+  _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______, _______,   _______  \
+)
+
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer